notify/aws/sns.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/megaease/easeprobe/global"
+	"github.com/megaease/easeprobe/probe"
+	log "github.com/sirupsen/logrus"
+)
+
+// SNSNotifyConfig is the AWS SNS notification configuration
+type SNSNotifyConfig struct {
+	Options  `yaml:",inline"`
+	TopicARN string          `yaml:"arn"`
+	client   *sns.SNS        `yaml:"-"`
+	Context  context.Context `yaml:"-"`
```

Comment:
Context needn't be exposed. Naming `context` is better.

Reply:
done!